lib/fauna/query.rb

```diff
@@ -680,6 +712,20 @@ def self.wrap_values(obj)
       def self.wrap_varargs(values)
         wrap(values.length == 1 ? values[0] : values)
       end
+
+      def self.wrap_write(ref, params)
```

Comment:
I don't think we need to wrap all write functions. We have `create_function` for better ergonomics. I'd say to add any necessary sugar there, instead.